lib/DocumentManagerInterface.php

```diff
@@ -60,7 +60,7 @@ public function remove($document);
      * Move the PHPCR node to which the document is mapped to be a child of the node at the given path or UUID.
      *
      * @param object $document
-     * @param string $destId Path or UUID
```

Comment:
Maybe we should change that to only path... The problem is that not every node has a UUID (e.g. or maybe only the newly created ones), but every node has a path (also the ones which are only existing in the current session).